x/ccv/provider/types/legacy_proposal.go

```diff
@@ -20,15 +20,16 @@ import (
 
 const (
 	ProposalTypeConsumerAddition     = "ConsumerAddition"
-	ProposalTypeConsumerRemoval      = "RemoveConsumer"
-	ProposalTypeConsumerModification = "UpdateConsumer"
+	ProposalTypeConsumerRemoval      = "ConsumerRemoval"
```

Comment:
Why the rename?

Reply:
I just reverted the entire file to what's in ICS main.